lib/vagrant/util/install_cli_autocomplete.rb

```diff
@@ -0,0 +1,65 @@
+module Vagrant
+  module Util
+    class Shell
+      PREPEND = "".freeze
+      STRING_INSERT = "".freeze
+      APPEND = "".freeze
+      CONFIG_PATHS = [""].freeze
+
+      def self.shell_installed(home)
+        self::CONFIG_PATHS.each do |path|
```

Comment:
I assume there is a better way to this `self::<variable>` bit. This is a gap in my ruby-foo, would appreciate a nudge in the right direction

Reply:
Maybe instead of these global constants you could have an `initialize` class and set them as class variables instead? Then you could simply access them with `@config_paths`, etc.

Reply:
And if we can get an initialize method, it would also be good to have a `logger` with `log4r` with this class. Then we can add some more extra context in the log around setting these up which will help with bug reports.